File: Easy/1710. Maximum Units on a Truck/1.py
class Solution:
    def maximumUnits(self, boxTypes: List[List[int]], truckSize: int) -> int:
        boxTypes.sort(key=lambda a:a[1],reverse=True)
        i=0
        r=0
        l=len(boxTypes)
        while i<l and truckSize>0:
            t=min(truckSize,boxTypes[i][0])
            r+=t*boxTypes[i][1]
            truckSize-=t
            i+=1
        return r

File: Easy/1. Two Sum/1.java
class Solution {
    public int[] twoSum(int[] n, int t) {
        for(int i=0,j;i<n.length;i++)for(j=i+1;j<n.length;j++)if(n[i]+n[j]==t)return new int[]{i,j};
        return null;
    }
}
